app/Models/Collection.php

```diff
@@ -479,6 +479,49 @@ public function recentlyViewed(): bool
         return $this->last_viewed_at->gte(now()->subDays(1));
     }
 
+    /**
+     * Determine whether the collection is *potentially* full. We call it "potentially" because there is no way for
+     * us to be 100% sure that the collection is full and we make our best guess.
+     */
+    public function isPotentiallyFull(): bool
+    {
+        // If there is no supply or if the NFTs for the collection haven't been indexed, we know for sure it's not full...
+        if ($this->last_indexed_token_number === null || $this->supply === null) {
```

Comment:
Please note that we are already filtering out collections with `supply=null`: https://app.clickup.com/t/85zu0hdha

It's worth double-checking to ensure that we are not excessively filtering out collections.

https://github.com/ArdentHQ/dashbrd/pull/101/files#diff-5254893bd34d47c136df16ce1c571ddd8d62d0bd9a0d1730edecbbb049fa7d89R52

Reply:
wdym? we're filtering collections where the supply is not null so this check (`$this->supply === null`) will always be false and therefore irrelevant in this case

Reply:
with the current way we call this `isPotentiallyFull` method we won't ever pass it a collection with a `null` supply, but for reliability I'd keep the check in case something calls it without the `withAcceptableSupply` scope as it avoids assumptions here